Synview-backend/middleware/token.ts

```diff
@@ -0,0 +1,16 @@
+import { Context } from "../deps.ts";
+export default async function Token(
+  context: Context,
+  next: () => Promise<unknown>
+) {
+  try {
+    const token = localStorage.getItem("token");
+    if (!token) {
+      return;
+    }
+    context.response.headers.set("Authorization", `Bearer ${token}`)
+    await next();
+  } catch (e) {
+    throw new Error("Didnt find any auth token: " + e);
+  }
+}
```

Comment:
same here. If you want, you might consolidate your JWT related logic like `getToken` and this `Token` function into some helper/util file. It is common to do something like `Synview-backend/utils/JWTHelpers.ts`